Anonymize/Type/FixedValueAnonymizer.php

```diff
@@ -54,6 +55,10 @@ public function anonymize($propertyValue, array $options = [])
         $this->configureOptions($resolver);
         $options = $resolver->resolve($options);
 
+        if ('' === $options['annotationValue']) {
```

Comment:
Shouldn't this be a `null` check? Why don't we allow empty strings on this one?

Reply:
yes, didn't get the chance to change it yet.